packages/indexer-agent/src/query-fees/allocations.ts

```diff
@@ -382,6 +382,8 @@ export class AllocationReceiptCollector implements ReceiptCollector {
         },
       )
     } catch (err) {
+      // seems to be trying repeatedly
+      // should this accumulate? or add attemps similar to querying proof of indexing?
```

Comment:
Interesting. 
1. Worth investigating why it's failing? 
2. Could probably improve logging as you suggest, so it's clear to the indexer how many attempts it's tried etc...

Reply:
Added attempts and gave a limit for 1000 for now... Though I can't seem to get indexer-agent to get to `submitVoucher` after I close allos.